qcodes/dataset/data_set.py

```diff
@@ -360,8 +360,9 @@ def add_results(self, results: List[Dict[str, VALUE]]) -> int:
 
         Args:
             - list of name, value dictionaries  where each
-              dictionary provides the values for all of the parameters in
-              that result.
+              dictionary provides the values for of the parameters in
```

Comment:
A small but important word is missing here.

Reply:
...or there is a word too many.

Reply:
I not sure i'm following....

Reply:
"dictionary provides the values for (all/none/some) of the parameters"
or, alternatively,
"dictionary provides the values for the parameters"